packages/aws-cdk-lib/aws-elasticloadbalancingv2/lib/shared/base-target-group.ts

```diff
@@ -64,6 +79,13 @@ export interface BaseTargetGroupProps {
    * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-elasticloadbalancingv2-targetgroup-targetgroupattribute.html
    */
   readonly crossZoneEnabled?: boolean;
+
+  /**
+   * The type of IP addresses of the targets registered with the target group.
+   *
+   * @default undefined - ELB defaults to IPv4
```

Comment:
I think `@default IPv4` is sufficient, but I have written it this way because several maintainers have recently recommended this approach.

- https://github.com/aws/aws-cdk/pull/32102#discussion_r1844420059
- https://github.com/aws/aws-cdk/pull/31988#discussion_r1830527931

Reply:
Thanks! I like it this way so that we know both the CDK default behaviour and CFN default behaviour.